app/assets/stylesheets/components/common/_social-sharing.scss

```diff
@@ -0,0 +1,34 @@
+.list {
+  > li.social-sharing__item:before {
+    content: none;
+  }
+}
+
+%icon {
+  .social-sharing & {
+    background-image: none;
+    display: none;
+  }
+}
+
+.icon {
+  .social-sharing & {
+    background-image: none;
+    display: none;
+  }
+}
+
+.social-sharing {
+  h2 {
+    @include body(24, 30);
+    color: $color-heading-extra-small;
+    margin-bottom: 0.5rem;
```

Comment:
We have a thing called `$baseline-unit` that ends up being outputted as `rem`'s or `em`'s depending on whether it's the responsive or fixed stylesheet. We tend to use it for vertical rhythm, so this is a good candidate for this.

You could use it like `margin-bottom: $baseline-unit`